Lombiq.Walkthroughs/Assets/Scripts/walkthroughs.js

```diff
@@ -0,0 +1,3770 @@
+            else {
+                if (!splitString) {
+                    splitString = '?';
+                }
+                else {
+                    splitString = new RegExp(splitString, 'i');
+                }
+
+                // eslint-disable-next-line prefer-destructuring
+                goToRelativePageString = window.location.href.split(splitString)[0];
+            }
+
+            const goToRelativePageURL = new URL(goToRelativePageString + (nextPage ?? ''));
+            goToRelativePageURL.searchParams.set('shepherdTour', shepherdTour);
+            goToRelativePageURL.searchParams.set('shepherdStep', shepherdStep);
+            window.location.href = goToRelativePageURL.toString();
+        }
+
+        ['complete', 'cancel'].forEach((event) => Shepherd.on(event, () => {
+            removeShepherdQueryParams();
+            deleteWalkthroughCookies();
+        }));
+
+        // Add new walkthroughs here.
+        const walkthroughs = {
+            orchardCoreAdminWalkthrough: new Shepherd.Tour({
+                // Tour id should be the same as the tour variable's name.
+                id: 'orchardCoreAdminWalkthrough',
+                useModalOverlay: true,
+                defaultStepOptions: {
+                    cancelIcon: {
+                        enabled: true,
+                    },
+                    when: {
+                        show() {
+                            addShepherdQueryParams();
+                        },
+                    },
+                },
+                steps: [
+                    {
+                        title: 'Welcome in the<br>Orchard Core Admin Walkthrough!',
+                        text: 'This walkthrough covers key Orchard Core features, such as content management, user roles, ' +
+                            'and theme selection, and points users to further learning resources.',
+                        buttons: [
+                            {
+                                action: function () {
+                                    removeShepherdQueryParams();
+                                    walkthroughs.orchardCoreAdminWalkthrough.complete();
+                                    // The walkthroughSelector is later defined and orchardCoreAdminWalkthrough is only used later.
+                                    // eslint-disable-next-line no-use-before-define
+                                    walkthroughSelector.start();
+                                },
+                                classes: 'shepherd-button-secondary',
+                                text: 'Back',
+                            },
+                            nextButton,
+                        ],
+                        id: 'welcome',
+                    },
+                    {
+                        title: 'Setup recipe',
+                        text: 'The setup recipe in <a href="https://github.com/Lombiq/Orchard-Walkthroughs">Lombiq.Walkthroughs module</a>' +
```

Comment:
All external links should be opened in a new window instead of using the current one. We have a module for that: Lombiq Helpful Extensions - Target Blank

Reply:
Lombiq Helpful Extensions - Target Blank is not working, I guess because how the walkthrough windows are popping up. So by the time a new one pops up, Lombiq Helpful Extensions - Target Blank is already done. Do you want me to add `target="_blank"` manually for the links?

Reply:
I mean it would be better that way. What do you think?

Reply:
Agree